lib/main.dart

```diff
@@ -84,60 +84,64 @@ void main() async {
     DartPingIOS.register();
   }
 
-  /// Allows the top-level notification handlers to trigger actions farther down
```

Comment:
Most of this logic was moved to `lib/notification/utils/local_notifications.dart`, including the background fetch task.